internal/server/server_test.go

```diff
@@ -16,6 +16,7 @@ import (
 
```

Comment:
This is the fold, nothing too interesting happens below. Test updates. And updating the service init, as mentioned above.